mesonbuild/environment.py

```diff
@@ -1087,7 +1087,7 @@ def sanitize(p):
                 return cls(
                     compiler, version, for_machine, is_cross, info, exe_wrap,
                     target, linker=linker)
-            if 'clang' in out:
+            if 'clang' in out or 'Clang' in out:
```

Comment:
Is the qualcom clang identical to vanilla clang (at least in terms of command line arguments)? If it's not we should do like we do with Apple's clang and have a separate representation.

Reply:
Yes it is. Qualcomm calls it the "baremetal" LLVM toolchain for Snapdragon chipsets.

Reply:
Yes it is. Qualcomm calls it the "baremetal" LLVM toolchain for Snapdragon chipsets.

Reply:
Okay, then this seems fine.